assets/src/domain/eventEditor/ui/ticketAssignmentsManager/TicketAssignmentsManager.tsx

```diff
@@ -1,13 +1,28 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 
 import ErrorMessage from './ErrorMessage';
 import { Table } from './table';
 import { TAMProps } from './types';
 import useTAMDatesAndTickets from './useTAMDatesAndTickets';
+import useTAMState from './useTAMState';
+import { useRelations } from '@appServices/apollo/relations';
 
 const TicketAssignmentsManager: React.FC<TAMProps> = ({ assignmentType, entityId }) => {
 	const datesAndTickets = useTAMDatesAndTickets({ assignmentType, entityId });
 
+	const relations = useRelations();
+	const { initialize, isInitialized } = useTAMState();
+
+	useEffect(() => {
+		if (!isInitialized()) {
+			// If TAM is only for a single datetime
+			// limit relations to that datetime
+			const forDate = assignmentType === 'forDate' ? entityId : null;
+			// initialize with existing data
+			initialize(relations.getData(), forDate);
+		}
+	}, [isInitialized()]);
```

Comment:
can this be executed in `useTAMState` and have just the result here?
it is a bit strange to see a function being executed as a dependency

Reply:
That's pretty normal. It's been defined as a function to avoid stale value getting found to a scope, this way i gets the current value to avoid the issue caused by asynchronous state updates.